src/Components/Alert/Components/Filters/__tests__/Medium.jest.tsx

```diff
@@ -0,0 +1,53 @@
+import { mount } from "enzyme"
+import {
+  AlertProvider,
+  useAlertContext,
+} from "Components/Alert/Hooks/useAlertContext"
+import { Medium } from "Components/Alert/Components/Filters/Medium"
+import {
+  ArtworkFilterContextProvider,
+  useArtworkFilterContext,
+} from "Components/ArtworkFilter/ArtworkFilterContext"
+
+jest.mock("Utils/Hooks/useMatchMedia", () => ({
+  __internal__useMatchMedia: () => ({}),
+}))
+
+describe("MediumFilter", () => {
+  let filterContext
+  let alertContext
+
+  const MediumFilterTestComponent = () => {
+    filterContext = useArtworkFilterContext()
+    alertContext = useAlertContext()
+
+    return <Medium />
+  }
+
+  const getWrapper = (contextProps = {}, initialCriteria = {}) => {
+    return mount(
+      <ArtworkFilterContextProvider {...contextProps}>
```

Comment:
**question**: Do we need this wrapping `ArtworkFilterContextProvider` component here? Unless we're specifically testing a component that makes use of the context (which shouldn't be the case with these new alert components), I don't think it adds much value in the test.

Reply:
I'm testing whether updating the alert filter affects the filter state of the `ArtworkFilterContext` [here](https://github.com/artsy/force/pull/13106/files#diff-157c5645f7fc64324b20c7393e897dab23f33efdeffdf304190ffd09ad467eb1R51) to make sure only the state in the alert context gets changed.

I thought it might be good to have it here because it's one of the requirements.

Reply:
Gothcha, I don't consider this blocking but in that case would it be possible to move this wrapping to only the test that exercises that requirement? Otherwise it feels a bit misleading to have it included for every test in the file.

I'd also support eventually moving this kind of assertion up a level once we're exercising a higher-level flow and not filter-specific — but also not blocking.

Reply:
This is a good argument. We don't need the assertion in every filter test. I removed it in https://github.com/artsy/force/pull/13112